controllers/follows/FollowController.ts

```diff
@@ -47,6 +52,7 @@ export default class FollowController implements IFollowController {
     const followerId: string = req.params.userId;
     const followeeId: string = req.body.followeeId;
 
+    const followNotification: Notification = await this.notificationDao.createNotificationForUser("FOLLOWS", followeeId, followerId);
```

Comment:
The follow should be created before the notification; hypothetically, if the follow creation fails, the notification should not be created.

Reply:
Just updated this.